### PR TITLE
Frontend issue#17 wireframes

### PR DESCRIPTION
**Pull request by AnthonyUch**
Built wireframes and included sample data to show how it might look once the api is set up. Click "Dashboard" in the navbar to navigate through the pages in the web browser.

### CODE REVIEW

**Comment by ajytan** [comment]
[April 2, 2021]
Pull request has been review by the Doc & Testing team

- Implementation matches the Figma
- Need to decide whether we are registering users with a _Username_ or _Email_ (Currently backend uses _Email_ while frontend uses _Username_)
- Functionality has not been implemented yet
- Pages currently just display some sample data
- No errors found

This pull request is ready to be merged into the "main" branch.

**Review by jakobfipke — COMMENTED** [review]
The pull request did not pass Github's automatic check for the deployment of the react app to azure. This must be fixed before I approve the pull request.

**Review by jakobfipke — APPROVED** [review]
Approved.

**Pull request merged**
